s2wav/tiling.py

```diff
@@ -101,7 +35,7 @@ def tiling_direction(L: int, N: int = 1) -> np.ndarray:
 
     ind = 1
```

Comment:
Previously we were iterating over ind, but now instead of `s_elm[ind]` we have `s_elm[el, L - 1 + m]`. I think it's safe to delete this line.

Reply:
Right you are, thanks!